website/docs/general/documentation-guide.md

```diff
@@ -0,0 +1,129 @@
+---
+id: documentation-guide
+title: Documentation style guide
+keywords:
+  - API gateway
+  - APISIX
+  - Apache APISIX
+  - project documentations
+description: Style guide for Apache APISIX documentation.
+---
+
+This document is a style guide for Apache APISIX documentation. This document is intended for contributors to APISIX's documentation and should be followed to ensure consistency.
+
+To learn more about contributing, see [Contributing flow](contributor-guide.md).
+
+## Tone, content and audience
+
+- Aim for a voice and tone that's conversational, friendly, and respectful without being frivolous. Learn more on [Voice and tone](https://developers.google.com/style/tone).
+- Write for accessibility. See [General do's and don'ts](https://developers.google.com/style/accessibility#general-dos-and-donts) to learn more.
+- Identify the intended audience and write for them. Find the common ground for you and your target audience.
+
+## Language
+
+- Use second person while writing unless it isn't appropriate.
+- Use active voice while writing.
+- Use standard American spellings.
+- Always place [conditional clauses before instructions](https://developers.google.com/style/clause-order).
+- Use the following spellings for project specific entities:
+  - Use Apache APISIX instead of APISIX when referencing the project in introductions and when referencing to the project community.
+  - Use APISIX instead when referencing to the project inside documentation.
+  - APISIX specific component names like Plugin and Route are always capitalized.
+  - Use correct acronyms when necessary. Use the expanded word in first usage and proceed to use the acronym in the rest of the references. For example:
+
+    | ✅               | ❌               |
+    | ---------------- | ---------------- |
```

Comment:
```suggestion
    | :--------------: | :--------------: |
```

For centering

Reply:
The normal `----` centers by default I guess.